tool/plugin/lib/plugin.dart

```diff
@@ -11,6 +11,16 @@ import 'package:path/path.dart' as p;
 
 import 'src/lint.dart';
 
+const plugins = const {
+  'io.flutter': '9212',
+  'io.flutter.as': '10139',
```

Comment:
Sounds like this is encoding the current ant build.xml functionality (and we'll likely move to one plugin ID, and building for specific platform version in the future: 2017.1, 2017.2, 2017.3).

Reply:
Exactly. I'll change this around when implementing `plugin build`. Including putting constants like this into a file.